daemon/src/wire.rs

```diff
@@ -137,3 +160,8 @@ impl From<CfdTransactions> for Msg1 {
         }
     }
 }
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Msg2 {
+    pub signed_lock: PartiallySignedTransaction,
```

Comment:
Ideally we would be using the psbt binary encoding here but that is not super important.

Reply:
I applied the same TODO as for `Msg0` 🤪